### DISCUSSION TITLE
message被截断的问题

### DISCUSSION THREAD

**Discussion by waterchi** [wechaty/wechaty#2563] [Q&A]
请问，当我在微信上发送消息，消息中有半角冒号:  ， 然后紧跟一个换行。在wechaty中收到的消息会丢失从冒号开始的所有内容。
从日志中看到，padlocal收到的消息还有冒号后的内容，比如：“aaa:\nbbb”，但是在wechaty的onmessage事件中收到的消息就只有"aaa"。
有人遇到过类似的问题么？应该如何解决呢？
谢谢。

**Comment by hmllsnow** [comment]
你好我也遇到这个问题了，请问您解决了吗？
